app/src/main/res/values-fr/strings.xml

```diff
@@ -147,6 +150,8 @@
     <string name="category_on_screen_controls_settings">Paramètres des contrôles à l\'écran</string>
     <string name="title_checkbox_show_onscreen_controls">Afficher les commandes à l\'écran</string>
     <string name="summary_checkbox_show_onscreen_controls">Afficher la superposition du contrôleur virtuel sur l\'écran tactile</string>
+    <string name="title_checkbox_vibrate_osc">Activer les vibrations</string>
+    <string name="summary_checkbox_vibrate_osc">Active l'émulation sur votre appareil des vibrations des commandes à l'écran</string>
```

Comment:
All `'` characters need to have `\` before them, like `\'`

Reply:
Sorry I had not seen you have the eye ;-)